src/rgw/rgw_rest_s3.cc

```diff
@@ -6376,15 +6545,17 @@ int RGWSelectObj_ObjStore_S3::send_response_data(bufferlist& bl, off_t ofs, off_
                         <<  " obj-size " << s->obj_size << dendl;
       continue; 
     }
-
+    bytesprocessed += it.length();
     status = run_s3select(m_sql_query.c_str(), &(it)[0], it.length());
     if(status<0) {
       break;
     }
     i++;
   }
+  m_aws_response_handler.get()->init_stats_response();
+  m_aws_response_handler.get()->send_stats_response(bytesprocessed,bytesprocessed,bytesprocessed);
+  m_aws_response_handler.get()->init_end_response();
```

Comment:
send_response_data is called many times (number of chunks)
thus, it's not the correct location for /stat response/, it should be one-time